framework/ui/src/components/organisms/vc-table/vc-table.vue

```diff
@@ -169,6 +168,42 @@
                   <vc-table-cell :cell="cell" :item="item"></vc-table-cell>
                 </slot>
               </td>
+              <td
+                class="vc-table__body-cell vc-table__body-cell_overflow"
+                width="44"
+                v-if="itemActionBuilder"
+              >
+                <div
+                  class="vc-table__body-actions-container vc-flex vc-flex-justify_center vc-flex-align_center"
+                >
+                  <div
```

Comment:
Need to show icon when there is at least one element generated by itemActionBuilder. Such check logic is implemented in mobile table view.